OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -80,6 +85,9 @@ void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData
 					if (fieldInfo.IsInitOnly)
 						continue;
 
+					if (alreadyLinted.Contains(fieldInfo.Name))
+						continue;
```

Comment:
This doesn't seem correct. Now the below code is redundant for anything but speeds

Reply:
I can also try catch the traits that are wrongly handled here, but that seemed worse. I also don't understandy how a skip line suddenly makes anything redundant. Please elaborate.

Reply:
the referencedKeys below has the exact same members as alreadyLinted except for speeds

Reply:
`alreadyLinted` contains the trait field names not the translation keys.

Reply:
ah, but doesn't it add multiple instances of the same field then? And also you should then add class name + field name to avoid incorrect overlap no?

Reply:
It also seems incorrect for yaml overwrites. The key can have both a default value and an overwrite value

Reply:
I agree that this is shit. The actual error was somewhere else.